2025/09/15 13:06:45 [warn] 830#830: *323838 an upstream response is buffered to a temporary file /var/cache/nginx/proxy_temp/0/00/0000139000 while reading upstream, client: 172.19.0.1, server: vec.org.br, request: "GET /download/40649 HTTP/1.1", upstream: "http://172.19.0.3:80/download/40649", host: "vec.org.br"
2025/09/15 15:50:56 [warn] 178#178: *63 an upstream response is buffered to a temporary file /var/cache/nginx/proxy_temp/1/00/0000000001 while reading upstream, client: 172.19.0.1, server: vec.org.br, request: "GET /download/40757 HTTP/1.1", upstream: "http://172.19.0.3:80/download/40757", host: "vec.org.br"
2025/09/15 16:08:26 [warn] 176#176: *162 an upstream response is buffered to a temporary file /var/cache/nginx/proxy_temp/2/00/0000000002 while reading upstream, client: 172.19.0.1, server: vec.org.br, request: "GET /download/40662 HTTP/1.1", upstream: "http://172.19.0.3:80/download/40662", host: "vec.org.br"
2025/09/20 20:37:45 [error] 186#186: *4828 open() "/data/letsencrypt-acme-challenge/.well-known/acme-challenge/2DHy3f8T1pc17oBe1a1xP0Cu-HuUWYbGk4PNOl9ZZjs" failed (2: No such file or directory), client: 172.18.0.1, server: vec.org.br, request: "GET /.well-known/acme-challenge/2DHy3f8T1pc17oBe1a1xP0Cu-HuUWYbGk4PNOl9ZZjs HTTP/1.1", host: "vec.org.br"
2025/09/20 22:37:35 [error] 186#186: *7480 open() "/data/letsencrypt-acme-challenge/.well-known/acme-challenge/xejVBbKZK91MBcaTy0G7OLSg2UfU0eTeJgDodI0frwM" failed (2: No such file or directory), client: 172.18.0.1, server: vec.org.br, request: "GET /.well-known/acme-challenge/xejVBbKZK91MBcaTy0G7OLSg2UfU0eTeJgDodI0frwM HTTP/1.1", host: "vec.org.br"
2025/09/20 23:37:46 [error] 303#303: *7822 open() "/data/letsencrypt-acme-challenge/.well-known/acme-challenge/3-qT0Tgm9aCL5WHGf-JgTHJb4c3_YemSypv7Ym9W3Lw" failed (2: No such file or directory), client: 172.18.0.1, server: vec.org.br, request: "GET /.well-known/acme-challenge/3-qT0Tgm9aCL5WHGf-JgTHJb4c3_YemSypv7Ym9W3Lw HTTP/1.1", host: "vec.org.br"
2025/09/21 00:37:35 [error] 303#303: *8155 open() "/data/letsencrypt-acme-challenge/.well-known/acme-challenge/njQqXDeOx7BCWZgKZRTxL045FwkIV30w4zd6j1Zunvc" failed (2: No such file or directory), client: 172.18.0.1, server: vec.org.br, request: "GET /.well-known/acme-challenge/njQqXDeOx7BCWZgKZRTxL045FwkIV30w4zd6j1Zunvc HTTP/1.1", host: "vec.org.br"
2025/09/21 01:37:46 [error] 303#303: *8494 open() "/data/letsencrypt-acme-challenge/.well-known/acme-challenge/v207PNJx1-kpQ4Q1OG6o3LN031KKrSOtQrEYt2MuMeY" failed (2: No such file or directory), client: 172.18.0.1, server: vec.org.br, request: "GET /.well-known/acme-challenge/v207PNJx1-kpQ4Q1OG6o3LN031KKrSOtQrEYt2MuMeY HTTP/1.1", host: "vec.org.br"
2025/09/21 02:03:42 [warn] 306#306: *8648 using uninitialized "server" variable while logging request, client: 172.18.0.1, server: vec.org.br, request: "GET / HTTP/1.1", host: "vec.org.br:443"
2025/09/21 02:37:51 [error] 303#303: *8843 open() "/data/letsencrypt-acme-challenge/.well-known/acme-challenge/62U_mUJgXW7XDbDw2e5sm1Qfa_Raa_6_sLHouK8n_ik" failed (2: No such file or directory), client: 172.18.0.1, server: vec.org.br, request: "GET /.well-known/acme-challenge/62U_mUJgXW7XDbDw2e5sm1Qfa_Raa_6_sLHouK8n_ik HTTP/1.1", host: "vec.org.br"
2025/09/21 07:37:47 [error] 312#312: *10458 open() "/data/letsencrypt-acme-challenge/.well-known/acme-challenge/JnIupI-dUU7FxvSoaISOyf_frTZuvEzX0ODfQPNhMUg" failed (2: No such file or directory), client: 172.18.0.1, server: vec.org.br, request: "GET /.well-known/acme-challenge/JnIupI-dUU7FxvSoaISOyf_frTZuvEzX0ODfQPNhMUg HTTP/1.1", host: "vec.org.br"
2025/09/21 08:37:39 [error] 312#312: *10779 open() "/data/letsencrypt-acme-challenge/.well-known/acme-challenge/6Arg-MCw9wC9BQyONjqMxJTWuopaMzAUmEPELC4icCQ" failed (2: No such file or directory), client: 172.18.0.1, server: vec.org.br, request: "GET /.well-known/acme-challenge/6Arg-MCw9wC9BQyONjqMxJTWuopaMzAUmEPELC4icCQ HTTP/1.1", host: "vec.org.br"
2025/09/21 09:37:44 [error] 312#312: *11079 open() "/data/letsencrypt-acme-challenge/.well-known/acme-challenge/GWAlvmDSlCzmkbCzzClr66sxUVRjpQc3bhaJUsl2fV4" failed (2: No such file or directory), client: 172.18.0.1, server: vec.org.br, request: "GET /.well-known/acme-challenge/GWAlvmDSlCzmkbCzzClr66sxUVRjpQc3bhaJUsl2fV4 HTTP/1.1", host: "vec.org.br"
2025/09/21 10:37:43 [error] 312#312: *11414 open() "/data/letsencrypt-acme-challenge/.well-known/acme-challenge/IlWexWtjVtKrr8iQNypGt4XWYNlocLOSSSEG2FGVH88" failed (2: No such file or directory), client: 172.18.0.1, server: vec.org.br, request: "GET /.well-known/acme-challenge/IlWexWtjVtKrr8iQNypGt4XWYNlocLOSSSEG2FGVH88 HTTP/1.1", host: "vec.org.br"
